### PR TITLE
Cindy expandable calendar

### PR DESCRIPTION
**Pull request by cindy1u0**
Added expandable feature and adjusted mobile styling
Screen Shot 2021-01-25 at 5 27 55 PM
Screen Shot 2021-01-25 at 5 27 34 PM
Screen Shot 2021-01-25 at 5 26 54 PM

### CODE REVIEW

**Comment by feliciazhang** [comment]
lgtm! only comment is nunito-sans? 🥺

**Comment by cindy1u0** [comment]
OmO I will change that

**Review by nupur-n — APPROVED** [review]
beauuuutiful! had a few styling thoughts (lining up the pills, the expanded view having the little color line like the calendar) but I feel like a secondary styling pass can be separate. 

looking good :D

File: src/components/Schedule.js [hunk 1]
```diff
@@ -39,81 +71,90 @@ const Schedule = () => {
...
+                                {width >= 1000 && (
+                                  <AdditionalAttributes event={event} />
+                                )}
+                              </div>
+                            </Card.Header>
+                            <Accordion.Collapse eventKey={idx.toString()}>
+                              <Card.Body>
+                                <strong>Description:&nbsp;</strong>
+                                {event.description}
+                                {width < 1000 && (
```
Comment (nupur-n) [inline src/components/Schedule.js:149]: is this here for small screens?

Comment (cindy1u0) [reply src/components/Schedule.js:149]: yep. I used 1000 because both 960 and 1200 didn't work super great on this

Comment (cindy1u0) [reply src/components/Schedule.js:149]: Right now technically we don't have the mobile version, so I am kinda free styling lol

**Review by snwessel — APPROVED** [review]
LGTM! I like that you included the little colored badges in the dropdown on mobile.

**Pull request merged**